dynamic-plugins/imports/package.json

```diff
@@ -34,6 +34,6 @@
     "@janus-idp/backstage-scaffolder-backend-module-regex": "1.4.12",
     "@janus-idp/backstage-scaffolder-backend-module-servicenow": "1.4.14",
     "@janus-idp/backstage-scaffolder-backend-module-sonarqube": "1.4.12",
-    "@janus-idp/backstage-plugin-catalog-backend-module-scaffolder-relation-processor": "1.0.3"
+    "@backstage-community/backstage-plugin-catalog-backend-module-scaffolder-relation-processor": "1.2.1"
```

Comment:
can you have a peer dependency from another repo?

Reply:
I think we can have a peer dependency from another repo to specify that our package relies on a specific version of another package